test/extended/util/manifests.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	o "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
+)
+
+func ReadFixture(path string) (runtime.Object, error) {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read file %q: %v", path, err)
+	}
+
+	obj, _, err := scheme.Codecs.UniversalDeserializer().Decode(data, nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return obj, nil
+}
+
+func ReadFixtureOrFail(path string) runtime.Object {
```

Comment:
`ReadFixtureOrDie` is the preferred name for these methods.

Reply:
thought that's the naming for cases when it raises `panic` this one just fails the tests

Reply:
In tests, that's basically that.